src/cljs/nr/gameboard/actions.cljs

```diff
@@ -33,10 +39,15 @@
   (when (= gameid (:gameid @game-state))
     (toast (tr [:game.inactivity "Game closed due to inactivity"]) "error" {:time-out 0 :close-button true})))
 
-(defmethod ws/-msg-handler :netrunner/state [{data :?data}] (init-game (parse-state data)))
+(defn handle-error []
+  (toast (tr [:game.error "Internal Server Error. Please type /bug in the chat and follow the instructions."]) "error" {:time-out 0 :close-button true})
+  (reset! lock false))
+
+(defmethod ws/-msg-handler :netrunner/state [{data :?data}] (reset-game (parse-state data)))
```

Comment:
When is init-game being called now?

Reply:
From launch-game, so as a result of :netrunner/start.